roles/node_exporter/meta/main.yml

```diff
@@ -0,0 +1,28 @@
+galaxy_info:
+  author: Michael Kaye
+  description: Deploys node_exporter from github
+  company: New Vector
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: BSD
+
+  min_ansible_version: 1.2
+
+  platforms:
+   - name: Debian
+     versions:
+     - jessie
```

Comment:
I'm not sure, but I *think* ansible want's a version number here, so 8. Also, any reason why this wouldn't run on stretch?

Reply:
More a statement of being untested on stretch

Reply:
Ahh okay